project_template/go.sh

```diff
@@ -8,22 +8,30 @@ EOF
 
 set -e
 
+# Detect if this script is run within an existing virtualenv and abort
+if [[ $VIRTUAL_ENV ]]; then
+    echo "ERROR: $0 will not work properly if run inside a venv; deactivate '`basename $VIRTUAL_ENV`'"
+    exit 1
+fi
```

Comment:
How does it not work? Do additional the other environment variables that get set when a virtualenv is activated interfere when you run a bin directly from another venv's bin directory? e.g. if I did `source foo/bin/activate` then `./bar/bin/python`, would it not just run `python` as if `bar` was activated?